spec/dummy/spec/lib/send_grid_mailer/definition_spec.rb

```diff
@@ -30,6 +30,30 @@
     end
   end
 
+  describe "#dynamic_template_data" do
+    before do
+      @substitution = subject.dynamic_template_data({ key: 'value' })
```

Comment:
Name your test subject if you need to reference it explicitly.<br>Redundant curly braces around a hash parameter.